scripts/windows_install_mingw64.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = "Stop"
+
+echo "starting mingw64 installation"
+
+curl.exe -L "https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe" -o msys2.exe
+if (!$?) { throw 'cmdfail' };
+
+.\msys2.exe -y -oC:\ ;
+if (!$?) { throw 'cmdfail' };
+Remove-Item msys2.exe ;
+
+function msys() { C:\msys64\usr\bin\bash.exe @('-lc') + @Args; if (!$?) { throw 'cmdfail' };} ; 
+msys ' ' ; 
```

Comment:
Add a comment explaining that this does the msys first-time setup tasks

Reply:
Sorry, what I meant was to add that to the msys '  '; bit. It's not obvious what that is intended to do otherwise.